mappings/net/minecraft/entity/ai/control/MoveControl.mapping

```diff
@@ -1,10 +1,12 @@
 CLASS net/minecraft/class_1335 net/minecraft/entity/ai/control/MoveControl
 	CLASS class_1336 State
 	FIELD field_6367 targetZ D
+	FIELD field_6368 forwardStrafe F
```

Comment:
This doesn't work, ["strafe" means to "move sideways"](https://en.wiktionary.org/wiki/strafe#Verb). Maybe `forwardMovement`?